android/app/src/main/res/values-en/strings.xml

```diff
@@ -195,4 +195,5 @@
     </string-array>
     <string name="label_miscellaneous_pref_category">Miscellaneous</string>
     <string name="label_shake_bug_reporter_invocation_method_pref">Shake to report a bug</string>
+    <string name="title_heuristique" translatable="false">L\'heuristique</string>
```

Comment:
Can you move the title below line 100 (in the `<!-- ÉTS -->` block)?

Reply:
Yes of course ! Sorry I used the translations editor 😅